CMakeModules/select_compute_arch.cmake

```diff
@@ -92,6 +92,23 @@ if(CUDA_VERSION VERSION_GREATER_EQUAL "11.1")
   set(CUDA_LIMIT_GPU_ARCHITECTURE "9.0")
 endif()
 
+if(CUDA_VERSION VERSION_GREATER_EQUAL "11.8")
```

Comment:
Was any architecture removed during these releases?

Reply:
Support for CC < 5.0 was removed in CUDA 12. I've updated the CUDA_ALL_GPU_ARCHITECTURES list.